plugin/src/arch/aarch64/ast.rs

```diff
@@ -24,53 +23,100 @@ pub struct RegVector {
     pub kind: RegKind,
     pub element_size: Size,
     pub lanes: Option<u8>,
-    pub element: Option<syn::Expr>
+    pub element: Option<syn::Expr>,
 }
 
 // Register id without indication of its usage. Either a static Regid or a family identifier + expression to choose the register
 #[derive(Debug, Clone)]
 pub enum RegKind {
     Static(RegId),
-    Dynamic(RegFamily, syn::Expr)
+    Dynamic(RegFamily, syn::Expr),
 }
 
 // a register identifier. This identifies an architecturally completely separate register.
 #[derive(Debug, PartialEq, Eq, Hash, Clone, Copy)]
 pub enum RegId {
     // regular registers. Either 4 or 8 bytes
-    X0 = 0x00, X1 = 0x01, X2 = 0x02, X3 = 0x03,
-    X4 = 0x04, X5 = 0x05, X6 = 0x06, X7 = 0x07,
-    X8 = 0x08, X9 = 0x09, X10= 0x0A, X11= 0x0B,
-    X12= 0x0C, X13= 0x0D, X14= 0x0E, X15= 0x0F,
-    X16= 0x10, X17= 0x11, X18= 0x12, X19= 0x13,
-    X20= 0x14, X21= 0x15, X22= 0x16, X23= 0x17,
-    X24= 0x18, X25= 0x19, X26= 0x1A, X27= 0x1B,
-    X28= 0x1C, X29= 0x1D, X30= 0x1E,
+    X0 = 0x00,
```

Comment:
And here is an example of it unrolling large enums for little reason. This doesn't make the file easier to read. dynasm-rs contains a significant amount of these tables, and the file really doesn't get more readable from multiple chunks of 32 lines of just register numbering.